### ISSUE TITLE
Other repo, `luakit_search_login`

### ISSUE THREAD

**Issue by o-jasper** [matthiasbeyer/pass2luakit#1]
Started changing, but ended up practically changing everything. [Here it is](https://github.com/o-jasper/luakit_search_login), what do you think? Could merge anyway.

Btw, `pass` is a bunch of shellscript.. Something simple implemented in go or nim or something might be preferable?

**Comment by matthiasbeyer** [comment]
This is a nice thing.

I'm currently not using luakit, but I plan to switch to it and I'd love a `pass` solution for this. I wouldn't implement a go or nim or anything variant, as pass works fine. No need to reimplement everything all the time!

I will test this as soon as possible, which might be in two weeks. As you might see I'm not that much of an lua guy, so your approach looks way better than mine, imho.

I'm undecided yet, but I guess we should continue with your approach (will have a closer look in said two weeks or so).

**Comment by o-jasper** [comment]
I agree, getting `pass` improved is very low priority. Main worry is that the input has some shellshock-like vulnerability. The domain name is sanitized already,(and might not have enough freedom in the first place) passwords are not.

Note that it uses [this pull request](https://github.com/mason-larobina/luakit/pull/269) that defines `load_asset`. Alternatively, the javascript can be put in a separate string after `local js=`. I prefer having it in a separate file.

I have already seen it not-work, due to not finding the forms.
